packages/veritone-widgets/src/widgets/MediaDetails/index.js

```diff
@@ -1442,19 +1455,20 @@ class MediaDetailsWidget extends React.Component {
                       )}
                     {!isImage &&
                       !isMedia &&
-                      !isPdf &&
-                      !!this.getPrimaryAssetUri() && (
+                      ((!isText && !!this.getPrimaryAssetUri()) ||
+                        (isText && !this.getPreviewUrl())) && (
                         <div className={styles.fileIconContainer}>
                           <InsertDriveFile className={styles.fileIcon} />
                         </div>
                       )}
                     {!isImage &&
```

Comment:
should leave the `PictureAsPdf` icon there for when a pdf is present, but the previewUrl has not been set yet